src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -1,33 +1,93 @@
-import * as React from "react";
-
+import React, { useState } from "react";
 import Checkbox, { CheckboxProps } from "./Checkbox";
 import { Story } from "@storybook/react/types-6-0";
 
+// Set defaults for the checkbox stories.
 export default {
     title: "Checkbox",
     component: Checkbox,
+    // Remove some props for all checkbox stories.
+    argTypes: {
+        checkboxId: {
+            table: {
+                disable: true,
+            },
+        },
+        onChange: {
+            table: {
+                disable: true,
+            },
+        },
+        labelOptions: {
+            table: {
+                disable: true,
+            },
+        },
+    },
 };
 
 const Template: Story<CheckboxProps> = (args) => <Checkbox {...args} />;
 
-export const BasicCheckbox = Template.bind({});
-BasicCheckbox.args = {
+// Uncontrolled checkbox.
+export const UncontrolledCheckbox = Template.bind({});
+
+// Uncontrolled checkbox set props
```

Comment:
Does it represent a breaking change to remove the uncontrolled checkbox entirely?  Controlled components are best practice, so it would be nice for us to encourage their usage by having the example only be things that are best practice.

Reply:
I think we can resolve this question as its covered below.